### PR TITLE
Savva Dzhumaliev Tasks 61-70

### PR DESCRIPTION
**Pull request by EasileleM**

### CODE REVIEW

File: tasks/63_insert-string/insert-string.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/**
+ * insert - inserts one string into another on given position
+ * If no insertable string is given, returns unchanged string
+ * @example
+ * // return 'We are doing some exercises.'
+ * insert('We are doing some exercises.')
```
Comment (vovcyan) [inline tasks/63_insert-string/insert-string.js]: Bad example

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/**
+ * smallestCommons - find the smallest common
+ * multiple of the provided parameters
```
Comment (vovcyan) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: What does it mean?

File: tasks/68_drop-elements/drop-elements.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/**
+ * dropElements - Drop the elements of an array,
+ * starting from the front, until the predicate returns true.
```
Comment (vovcyan) [inline tasks/68_drop-elements/drop-elements.js]: Incorrect "starting from the front"

File: tasks/69_unpack-elements/unpack-elements.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/**
+ * unpackArray - flatten a nested array
```
Comment (vovcyan) [inline tasks/69_unpack-elements/unpack-elements.js]: Incorrect description

File: tasks/70_binary-decode/binary-decode.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+/**
+ * binaryDecode - return an English translated sentence
```
Comment (vovcyan) [inline tasks/70_binary-decode/binary-decode.js]: Incorrect description

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by akimovaa — APPROVED** [review]

**Review by vovcyan — APPROVED** [review]

**Pull request merged**